websites/M/Mavanimes/dist/metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "service": "Mavanimes",
+  "author": {
+    "name": "Adrien",
+    "id": "256458960666624003"
+  },
+  "category": "anime",
+  "logo": "https://i.imgur.com/PgicAOE.png",
+  "thumbnail": "https://i.imgur.com/XQddkkZ.png",
```

Comment:
thumbnail should ideally be a screenshot of the website's homepage, not what you currently have